src/pages/concepts/cicd.mdx

```diff
@@ -0,0 +1,35 @@
+# Your CI/CD Pipeline with Nitric
+
+Nitric integrates seamlessly into CI/CD pipelines to abstract and automate infrastructure management. It’s an action that can slot into your workflow much like any other CLI.
+
+Familiar CI/CD tools like GitLab CI and GitHub Actions work great with Nitric. With hooks for pre and post actions, workflows may continue to leverage tools that ensure best practices and standards are met: security scans, linting, performance monitoring, compliance checks, etc. Nitric also supports various cloud environments and IaC tools so you have flexibility in your deployment.
+
+Here’s a typical deployment pipeline with Nitric:
+
+![diagram](/docs/images/concepts/ci-cd.svg)
+
+## Build
+
+Developers commit their application code to a code repository and utilize the Nitric SDK to manage cloud resources. This SDK simplifies the resource management process by abstracting infrastructure complexities, allowing developers to focus on coding. This ensures consistent resource configurations across environments, reducing the risk of configuration drift and speeding up the development process.
+
+## Commit
+
+Your Nitric application code is committed as usual with Git to take advantage of your version control and rollback capabilities. However, unlike traditional processes, the resource specification and infrastructure code isn’t checked in; Nitric generates those in real-time when a deployment is triggered.
+
+## Trigger
+
+Nitric slots in with the pre and post-actions that you currently have configured in your deployment pipeline.
+
+For example, you may have pre-actions in place such as application unit tests, other tools/scans, and IaC module unit tests.
+
+Once those are complete, you’ll trigger [deployment with the Nitric CLI](/concepts/ops-workflow). This will automate the creation of resource specifications, container images, and deployment scripts. Deployment scripts are generated by mapping resource requirements from the specification with IaC providers like Pulumi or Terraform. Depending on the provider choice, resources can be immediately provisioned with Nitric or you can output IaC for tools like Terraform.
```

Comment:
```suggestion
Once those are complete, you’ll trigger [deployment with the Nitric CLI](/concepts/how-ops-use-nitric). This will automate the creation of resource specifications, container images, and deployment scripts. Deployment scripts are generated by mapping resource requirements from the specification with IaC providers like Pulumi or Terraform. Depending on the provider choice, resources can be immediately provisioned with Nitric or you can output IaC for tools like Terraform.
```